core/modules/update/update.compare.inc

```diff
@@ -194,24 +197,26 @@ function _update_process_info_list(&$projects, $list, $project_type, $status) {
     // or theme. If the project name is 'backdrop', we don't want it to show up
     // under the usual "Modules" section, we put it at a special "Core"
     // section at the top of the report.
-    if ($project_name == 'backdrop') {
-      $project_display_type = 'core';
-    }
-    else {
-      $project_display_type = $project_type;
-    }
+if ($project_name == 'backdrop') {
```

Comment:
Not relevant for functionality, but I think this indentation change was unintentional. :wink:

Reply:
It is intentional 🙂 ...so that I know to return to this bit of code for some additional cleanup. It will be fixed during the final polishing of this PR.

Reply:
Ah, it was a "marker", got it. :+1:

Reply:
This one was reverted. I'll implement the clean-up I meant to do in a follow-up.